microsoft-365/enterprise/session-timeouts.md

```diff
@@ -40,7 +40,7 @@ The following table lists the session lifetimes for Microsoft 365 services:
 |Microsoft 365 admin center  <br/> |You are asked to provide credentials for the admin center every 8 hours.  <br/> |
 |SharePoint Online  <br/> |5 days of inactivity as long as the users chooses **Keep me signed in**. If the user accesses SharePoint Online again after 24 or more hours have passed from the previous sign-in, the timeout value is reset to 5 days.  <br/> |
 |Outlook Web App  <br/> |6 hours.  <br/> You can change this value by using the  _ActivityBasedAuthenticationTimeoutInterval_ parameter in the [Set-OrganizationConfig](https://go.microsoft.com/fwlink/p/?LinkId=615378) cmdlet.  <br/> |
-|Azure Active Directory  <br/> (Used by Office 2013 Windows clients with modern authentication enabled)  <br/> | Modern authentication uses access tokens and refresh tokens to grant user access to Microsoft 365 resources using Azure Active Directory. An access token is a JSON Web Token provided after a successful authentication and is valid for 1 hour. A refresh token with a longer lifetime is also provided. When access tokens expire, Office clients use a valid refresh token to obtain a new access token. This exchange succeeds if the user's initial authentication is still valid.  <br/>  Refresh tokens are valid for 90 days, and with continuous use, they can be valid until revoked.  <br/>  Refresh tokens can be invalidated by several events such as :  <br/>  User's password has changed since the refresh token was issued.  <br/>  An administrator can apply conditional access policies which restrict access to the resource the user is trying to access.  <br/> |
+|Azure Active Directory  <br/> (Used by Office and M365 applications in Windows clients with modern authentication enabled)  <br/> | Modern authentication uses access tokens and refresh tokens to grant user access to Microsoft 365 resources using Azure Active Directory. An access token is a JSON Web Token provided after a successful authentication and is valid for 1 hour. A refresh token with a longer lifetime is also provided. When access tokens expire, Office clients use a valid refresh token to obtain a new access token. This exchange succeeds if the user's initial authentication is still valid.  <br/>  Refresh tokens are valid for 90 days, and with continuous use, they can be valid until revoked.  <br/>  Refresh tokens can be invalidated by several events such as :  <br/>  User's password has changed since the refresh token was issued.  <br/>  An administrator can apply conditional access policies which restrict access to the resource the user is trying to access.  <br/> |
```

Comment:
```suggestion
|Azure Active Directory  <br/> (Used by Office and Microsoft 365 applications in Windows clients with modern authentication enabled)  <br/> | Modern authentication uses access tokens and refresh tokens to grant user access to Microsoft 365 resources using Azure Active Directory. An access token is a JSON Web Token provided after a successful authentication and is valid for 1 hour. A refresh token with a longer lifetime is also provided. When access tokens expire, Office clients use a valid refresh token to obtain a new access token. This exchange succeeds if the user's initial authentication is still valid.  <br/>  Refresh tokens are valid for 90 days, and with continuous use, they can be valid until revoked.  <br/>  Refresh tokens can be invalidated by several events such as:  <br/>  User's password has changed since the refresh token was issued.  <br/>  An administrator can apply conditional access policies that restrict access to the resource the user is trying to access.  <br/> |
```

Reply:
Three minor edits as indicated:
1. Please change "M365" to "Microsoft 365" in the first line.
2. Remove the extra space before the colon in the second last line ("... several events such as:").
3. Change "... conditional access policies which restrict ..." to "... conditional access policies that restrict ...).